virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -40,3 +43,32 @@ def create_disk(disk_type, path=None, size="500M", disk_format="raw", extra='',
     else:
         return libvirt.create_local_disk(disk_type, path=path, size=size,
                                          disk_format=disk_format, extra=extra)
+
+
+def create_primitive_disk(type_name, disk_device, device_target, device_bus,
+                          device_format, disk_src_dict, disk_auth_dict):
+    """
+    Creates primitive disk xml
+    :param type_name: disk type
+    :param disk_device: disk device
+    :param device_target: target device
+    :param device_bus: device bus
+    :param device_format: device format
+    :param disk_src_dict: disk source
+    :param disk_auth_dict: disk auth information
```

Comment:
I think above 2 need to give explanation on what type of dict it should give, but not the normal dict

Reply:
I add one example in doc string